### PR TITLE
[HOTFIX][4.5] Logging Warning->Info when PerfConfigs from db can't be deserialized or invalid (SWDEV-288975)

### PR DESCRIPTION
**Pull request by JehandadKhan**

### CODE REVIEW

File: src/include/miopen/db_record.hpp [hunk 1]
```diff
@@ -223,8 +223,8 @@ class DbRecord
 
         const bool ok = values.Deserialize(s);
         if(!ok)
-            MIOPEN_LOG_WE(
-                "Perf db record is obsolete or corrupt: " << s << ". Performance may degrade.");
+            MIOPEN_LOG_I("Perf db record is obsolete or corrupt: " << s
```
Comment (atamazov) [inline src/include/miopen/db_record.hpp:226]: If you follow [this recommendation](https://github.com/ROCmSoftwarePlatform/MIOpen/pull/1214#issuecomment-938204105), then please use `MIOPEN_LOG_IE` (info message for release builds, error for DEV or debug builds).

**Review by atamazov — APPROVED** [review]
Please go ahead if this works for you as is.

**Comment by atamazov** [comment]
@JehandadKhan Please let me know if you need more assistance; I can polish this PR for you.

**Comment by atamazov** [comment]
I'll look here tomorrow SPB morning and make changes if necessary ;)

**Comment by junliume** [comment]
> > @JehandadKhan This should be merged into staging branch, isn't it?
> 
> @junliume @JehandadKhan If you want to merge directly to the release branch, then please cherry-pick the same also into staging -- let's keep both in sync, so we'll be able to add more commits to staging and merge into release, if need arise.

Done for both release/rocm-rel-4.5-staging-verified (the one actually merged into release/rocm-rel-4.5) and release/rocm-rel-4.5-staging (the one has DB updates to remove solver 2c73a4f28801dec087d7117f7ffcb789bf026f71)

**Comment by atamazov** [comment]
@junliume 
> Done for ... release/rocm-rel-4.5-staging (the one has DB updates to remove solver 2c73a4f28801dec087d7117f7ffcb789bf026f71)

I just cherry-picked #1211 is into `release/rocm-rel-4.5-staging`. Now 4.5 release staging has W/A for SWDEV-305815 identical to the one that resides in `develop` (the most "valid" one!), ready for promotion into 4.5 release, just in case if need arise.

**Pull request merged**